spdmlib/src/responder/key_exchange_rsp.rs

```diff
@@ -516,7 +516,7 @@ mod tests_responder {
                 .ok()
                 .unwrap();
         let public_key_old = private_key.compute_public_key().ok().unwrap();
-        let public_key = BytesMut::from(&public_key_old.as_ref()[1..]);
+        let public_key = BytesMutStrubbed::from(&public_key_old.as_ref()[1..]);
```

Comment:
you dont need this for public_key

Reply:
> you dont need this for public_key

Yes, updated. So now the struct use BytesMut are SpdmDigestStruct, SpdmSignatureStruct and SpdmDheExchangeStruct.
The struct use BytesMutStrubbed are SpdmHKDFKeyStruct, SpdmDheFinalKeyStruct, SpdmAeadKeyStruct and SpdmAeadIvStruct.